Cargo.toml

```diff
@@ -45,6 +45,7 @@ actix-files = "0.5"
 actix-multipart = "0.3.0"
 actix-web-httpauth = "0.5.0"
 mime = "0.3"
+httparse = "1.3.5"
```

Comment:
Oh, forgot to mention, but since this is a version 1 dep, please change it depend on `"1"`. Since actix-http depends on hyper which depends on its own version of httparse, giving the versioning more leniency allows us to use the same version in both cases which decreases binary size.

Reply:
Ok. I will fix this.